Makefile.am

```diff
@@ -229,6 +233,15 @@ EXTRA_DIST += \
 	nmsg/base/pkt.c \
 	nmsg/base/xml.c
 
+FLT_LIBTOOL_FLAGS = -module -avoid-version -shared -export-symbols-regex "^nmsg_fltmod_plugin_export$$"
```

Comment:
I'm planning on building a separate `sie-nmsg-fltmod` repository that can serve as a fully worked standalone reference implementation. (Similarly to how `sie-nmsg` provides a standalone reference for `msgmod_plugin`. Probably `sie-nmsg` should be renamed to `sie-nmsg-msgmod`.) I might also write a blog post.

As far as just "how to use automake to build plugins" in general, that's covered pretty well by existing sources like the excellent [Autotools Mythbuster](https://autotools.io/libtool/plugins.html).

Do you think there should actually be a document in-tree in the `nmsg` repository describing how to build a standalone filter module plugin?